gatsby-config.js

```diff
@@ -112,6 +112,27 @@ const plugins = [
       },
     },
   },
+  {
+    resolve: "gatsby-plugin-git-clone",
+    options: {
+      repository: "https://github.com/wp-graphql/wp-graphql.git",
+      path: `${__dirname}/documentation/wp-graphql/main`,
+      branch: "master",
+    },
+  },
+  {
+    resolve: "gatsby-source-filesystem",
+    options: {
+      path: `${__dirname}/documentation/wp-graphql/main/docs`,
```

Comment:
If the /docs/ dir doesn't exist, which is doesn't right now, then I've seen this fail because the dir doesn't exist in the filesystem.  So, maybe we'll have to add first doc to wp-graphql repo master branch first.